server/consumer.go

```diff
@@ -1055,15 +1056,19 @@ func (o *consumer) deleteNotActive() {
 			// Check to make sure we went away.
 			// Don't think this needs to be a monitored go routine.
 			go func() {
+				var fs bool
 				ticker := time.NewTicker(time.Second)
 				defer ticker.Stop()
 				for range ticker.C {
 					js.mu.RLock()
 					ca := js.consumerAssignment(acc, stream, name)
 					js.mu.RUnlock()
-					if ca != nil {
-						s.Warnf("Consumer assignment not cleaned up, retrying")
-						meta.ForwardProposal(removeEntry)
+					if ca != nil && fs {
+						if fs {
+							s.Warnf("Consumer assignment not cleaned up, retrying")
+							meta.ForwardProposal(removeEntry)
+						}
```

Comment:
the if on 1066 already checks if `fs` is true, then 1067 again checks if `fs` is true and 1071 in a block where `fs` is always `true` sets it to `true`

Reply:
Yea that one got messed up after several revisions, will fix.

Reply:
ok fixed.

Reply:
Actually redid this again, sorry. I had it mostly right before. Please take one more look, apologies.